python/llm/src/bigdl/llm/transformers/convert.py

```diff
@@ -364,7 +365,10 @@ def ggml_convert_low_bit(model, qtype, optimize_model=True,
             "an issue on github if you think this is a bug."
         )
     elif device == "cpu":
-        if not (getattr(model, "quantization_method", None) == "gptq"):
+        if not (getattr(model, "quantization_method", None) == "gptq") and \
+            qtype not in [ggml_tensor_qtype["bf16"],
+                          ggml_tensor_qtype["fp16"]]:
```

Comment:
This can be potentially problematic if running on CPU (which may not support fp16/bf16)

Reply:
Yes, but on CPU which not support fp16/bf16, I guess users just should not use `torch_dtype=bf16/fp16` or `load_in_low_bit=bf16/fp16`, otherwise, they will meet similar error or perf drop.
Do you think we should handle this case (maybe add warning or raise error in forward of `BF16Linear`) ? Not sure whether this make sense and can really capture errors.